src/run.js

```diff
@@ -78,23 +80,22 @@ export function createProvideWrapper({
     const state = stateSubject.value;
     const patch = transitionFn(state);
 
-    // log('state is:', state);
-    // log('patch is:', patch);
+    log('state is:', state);
+    log('patch is:', patch);
     const shouldReplace = isTransitionFunction;
-    const newState = stateType(pickBy(
-      transitionReducer(shouldReplace ? { ...patch } : { ...state, ...patch }),
-      x => !t.Nil.is(x)
-    ));
+    const newState = stateType(
+      transitionReducer(shouldReplace ? { ...patch } : { ...state, ...patch })
```

Comment:
oh right, it's passed as an argument by the other function (I thought it was used before)
👍